powersimdata/data_access/profile_helper.py

```diff
@@ -30,22 +29,8 @@ def get_profile_version_cloud(grid_model, kind):
         *'demand_flexibility_cost_up'*, or *'demand_flexibility_cost_dn'*.
     :return: (*list*) -- available profile version.
     """
-    bfs = fs.open_fs("azblob://besciences@profiles").opendir(f"raw/{grid_model}")
-    return _get_profile_version(bfs, kind)
-
-
-def get_profile_version_local(grid_model, kind):
-    """Returns available raw profile from local file.
-
-    :param str grid_model: grid model.
-    :param str kind: *'demand'*, *'hydro'*, *'solar'*, *'wind'*,
-        *'demand_flexibility_up'*, *'demand_flexibility_dn'*,
-        *'demand_flexibility_cost_up'*, or *'demand_flexibility_cost_dn'*.
-    :return: (*list*) -- available profile version.
-    """
-    profile_dir = fs.path.join(server_setup.LOCAL_DIR, "raw", grid_model)
-    lfs = fs.open_fs(profile_dir, create=True)
-    return _get_profile_version(lfs, kind)
+    bfs = fs.open_fs("azblob://besciences@profiles")
+    return get_profile_version(bfs, grid_model, kind)
```

Comment:
Is there a reason this still needs to be its own function? Could the contents of `bfs` just be passed into `get_profile_version` in the `blob_version` located in `data_access.py`?

Reply:
Good call, done.